From vapier@gentoo.org  Tue Jun  1 13:32:06 2010
From: Mike Frysinger <vapier@gentoo.org>
Date: Fri, 21 May 2010 04:37:42 -0400
Subject: USB: isp1362: fix inw warning on Blackfin systems
To: linux-usb@vger.kernel.org, Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <1274431062-22069-1-git-send-email-vapier@gentoo.org>


The Blackfin code is incorrectly casting the argument to inw() to a pointer.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/isp1362.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/host/isp1362.h
+++ b/drivers/usb/host/isp1362.h
@@ -65,7 +65,7 @@ static inline void delayed_insw(unsigned
 	unsigned short *bp = (unsigned short *)buf;
 	while (len--) {
 		DUMMY_DELAY_ACCESS;
-		*bp++ = inw((void *)addr);
+		*bp++ = inw(addr);
 	}
 }
 
